src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -1,30 +1,109 @@
-import { GraphRow } from '@axosoft/gitkraken-components/lib/components/graph/GraphContainer';
+import GraphContainer, {
+	CssVariables,
+	GraphRow,
+	Head,
+	Remote,
+	Tag
+} from '@axosoft/gitkraken-components/lib/components/graph/GraphContainer';
 import React, { useEffect, useState } from 'react';
 import {
 	CommitListCallback,
+	GitBranch,
 	GitCommit,
+	GitRemote,
+	GitTag,
 	GraphColumnConfig,
 	Repository,
 	State,
 } from '../../../../plus/webviews/graph/protocol';
-import { GKGraph } from './GKGraph';
 
 export interface GraphWrapperProps extends State {
+	nonce?: string;
 	subscriber: (callback: CommitListCallback) => () => void;
 	onSelectRepository?: (repository: Repository) => void;
 	onColumnChange?: (name: string, settings: GraphColumnConfig) => void;
-	nonce?: string;
 	onMoreCommits?: (limit?: number) => void;
 }
 
-// TODO: this needs to be replaced with a function from the Graph repo
-const getGraphModel = (data: GitCommit[] = []) => data;
+// Copied from original pushed code of Miggy E.
+// TODO: review that code as I'm not sure if it is the correct way to do that in Gitlens side.
+// I suppose we need to use the GitLens themes here instead.
+export const getCssVariables = (): CssVariables => {
+    const body = document.body;
+    const computedStyle = window.getComputedStyle(body);
+    return {
+        '--app__bg0': computedStyle.getPropertyValue('--color-background'),
+        // note that we should probably do something theme-related here, (dark theme we lighten, light theme we darken)
+        '--panel__bg0':computedStyle.getPropertyValue('--color-background--lighten-05'),
+    };
+};
```

Comment:
this is loosely based off of what gitlens was doing with their theming and usage of vscode here: https://github.com/gitkraken/vscode-gitlens/blob/main/src/webviews/apps/shared/theme.ts